packages/discord.js/typings/index.d.ts

```diff
@@ -6351,15 +6354,23 @@ export interface InteractionCollectorOptions<
 }
 
 export interface InteractionDeferReplyOptions {
+  /** @deprecated Use {@link InteractionDeferReplyOptions.flags} instead. */
   ephemeral?: boolean;
+  flags?: BitFieldResolvable<
+    Extract<MessageFlagsString, 'Ephemeral' | 'SuppressEmbeds' | 'SuppressNotifications'>,
+    MessageFlags.Ephemeral | MessageFlags.SuppressEmbeds | MessageFlags.SuppressNotifications
+  >;
```

Comment:
Still not ideal to have this entire thing show up in the docs, but it can be address in another pull request (if at all)